app/controllers/application_controller.rb

```diff
@@ -1,8 +1,25 @@
 class ApplicationController < ActionController::Base
   include Authenticate
 
+  def default_render
+    if api_request?
+      json_payload = user_defined_ivars.map { |i| [ i.to_s[1..], instance_variable_get(i) ] }.to_h
+      render json: json_payload, status: :ok
+    else
+      super
+    end
+  end
+
   private
 
+  def api_request?
+    request.format.json?
+  end
+
+  def user_defined_ivars
```

Comment:
There were two ivars that I didn't expect to see - `rendered_format` and `marked_for_same_origin_verification` - but otherwise it seemed to work great!

Reply:
Good point. I have an idea for how to detect which ivars are created from rails internals and exclude those

Reply:
I figured out a way to ONLY include user created instance variables in the response. A simple before_action to remove anything that existed before the action ran.

Reply:
```
{
  "system_ivars": [
    "@rendered_format",
    "@marked_for_same_origin_verification"
  ],
```
😂 - we'll need to remove `@system_ivars` from the list as well, but it worked great otherwise!

Reply:
Ack! Yes :) Done